tests/operator_test.go

```diff
@@ -637,7 +637,9 @@ var _ = Describe("[Serial][sig-operator]Operator", func() {
 
 			vmis := []*v1.VirtualMachineInstance{}
 			for i := 0; i < num; i++ {
-				vmis = append(vmis, tests.NewRandomVMIWithEphemeralDisk(cd.ContainerDiskFor(cd.ContainerDiskCirros)))
+				vmi := tests.NewRandomVMIWithEphemeralDisk(cd.ContainerDiskFor(cd.ContainerDiskCirros))
+				tests.AddCloudInitConfigDriveData(vmi, "configdrive", "user test", "network test", false)
```

Comment:
maybe we should add all the generated disks here. For example, as we do in https://github.com/kubevirt/kubevirt/blob/main/tests/migration_test.go#L2018

Reply:
Yes, will do!
Note: we can't have 2 cloud init drives, that breaks the VMI (I'll file a bug eventually), so we just have to pick between no-cloud and config-drive.